CONTRIBUTING.md

```diff
@@ -18,3 +18,13 @@ run `make test` from root directory to check the updated template.
 
 [1]: https://github.com/docker-science/cookiecutter-docker-science/issues/new
 [2]: https://github.com/docker-science/cookiecutter-docker-science/compare
+
+
+## Target Users
+ - Having knowledge the basics of how to use the Docker
```

Comment:
starting from `having` is not natural English. "Basic knowledge on how to ~~~" would be natural. The following is an sample. 

- Basic knowledge of Docker environments
  - relationship between Docker images and containers
  - Docker container lifecycle
-  Experience with Makefile

Reply:
Thanks reviewing. I will fix it soon.